babel.config.json

```diff
@@ -16,7 +16,7 @@
       "@babel/plugin-transform-runtime",
       {
         "useESModules": true,
-        "regenerator": false
+        "regenerator": true
```

Comment:
I think default value is true

Reply:
Generated the config with the single-spa CLI:
```create-single-spa --moduleType util-module --framework react```
🤷